scripts/api.mjs

```diff
@@ -53,6 +53,14 @@ const pluginApis = [
       'https://github.com/ionic-team/capacitor-background-runner/blob/main/packages/capacitor-plugin/src/definitions.ts',
     tag: 'latest',
   },
+  {
+    id: 'barcode-scanner',
+    isCore: false,
+    isExperimental: false,
+    npmScope: '@capacitor',
+    editUrl: 'https://github.com/ionic-team/capacitor-barcode-scanner/blob/main/README.md',
```

Comment:
this is the root README of the repository, shouldn't it point to the plugin's README?
The repo seems to copy the plugins README to the root, so if users make changes on the root README they will be overwritten